paper.tex

```diff
@@ -107,20 +108,48 @@ \section{Trading Model}
 In the following, we describe the data trading process in detail. To participate a data marketplace, data providers and consumers have to register first. Then data provider can launch its product on the marketplace. Once a product is launched, it is searchable and could be traded afterward. The whole trading and refunding process is defined in smart contracts which are easily traceable and irreversible.
 
 \subsection{Participants Registration}
-At the beginning, the registrar creates a Registration Contact, which maintains all participants' information, including their DID (Decentralized Identifiers) information and public keys. Then some preselected brokers who are trustworthy are added to the decentralized data marketplace. Anyone who would like to sell or purchase data may register to become the data providers or consumers. The registrar has the authority to agree with applications.
+At the beginning, the registrar creates a Registration Contact (Figure \ref{fig:registration_contract}), which maintains all participants' information, including their DID (Decentralized Identifiers) information and public keys. Then some preselected brokers who are trustworthy are added to the decentralized data marketplace. Anyone who would like to sell or purchase data may register to become the data providers or consumers. The registrar has the authority to agree with applications.
```

Comment:
We shall not refer to "DID information". Use "DID Document" instead.

Reply:
I am not sure if "preselected brokers" is a meaningful term. How about something like Committee of Conformity Assessment?